alcali/package/install.sls

```diff
@@ -69,11 +69,27 @@ alcali-package-install-virtualenv-managed:
     - require:
       - git: alcali-package-install-git-latest
 
-alcali-package-install-pip-installed:
+alcali-package-install-db-pip-installed:
   pip.installed:
     - name: {{ db_connector }}
     - user: {{ alcali.deploy.user }}
     - cwd: {{ alcali.deploy.directory }}
     - bin_env: {{ alcali.deploy.directory }}/.venv
     - require:
       - virtualenv: alcali-package-install-virtualenv-managed
+
+{% if alcali.config.auth_backend == 'ldap' %}
```

Comment:
If auth_backend is not defined, then the state will fail to compile:

`    Rendering SLS 'base:alcali.package.install' failed: Jinja variable 'dict object' has no attribute 'auth_backend'`